### PR TITLE
Update ory.sh config

### PR DESCRIPTION
**Pull request by aeneasr**
# Pull request motivation(s)

We switched to docusaurus v2 and also separated the docs for the projects which is why this needs to be updated.

### What is the current behaviour?

Not relevant imo

### What is the expected behaviour?

Not relevant imo

##### NB: Do you want to request a **feature** or report a **bug**?

Not relevant imo

##### NB2: Any other feedback / questions ?

Not relevant imo

### CODE REVIEW

File: configs/ory.json [hunk 1]
```diff
@@ -4,43 +4,73 @@
...
       ]
     }
   ],
   "sitemap_urls": [
-    "https://ory.sh/docs/sitemap.xml"
+    "https://www.ory.sh/docs/sitemap.xml",
+    "https://www.ory.sh/hydra/docs/sitemap.xml",
+    "https://www.ory.sh/oathkeeper/docs/sitemap.xml",
+    "https://www.ory.sh/keto/docs/sitemap.xml",
+    "https://www.ory.sh/kratos/docs/sitemap.xml",
```
Comment (s-pace) [inline configs/ory.json:46]:
```suggestion
    "https://www.ory.sh/kratos/docs/sitemap.xml"
```

**Comment by aeneasr** [comment]
Whoops sorry for the typo! Thanks!

**Comment by aeneasr** [comment]
Is there any way for me to debug the facet filters or know when the index has updated? Right now, search doesn't work at all any more for me when using the tags/facets

**Comment by s-pace** [comment]
Hi there,

Congratulations, your search is now ready!
I've successfully configured the underlying crawler and it will now run every 24h.

You're now a few steps away from having it working on your website:

- Copy the following CSS/JS snippets and add them to your page


<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />


<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({
  apiKey: '8463c6ece843b377565726bb4ed325b0',
  indexName: 'ory',
  inputSelector: '### REPLACE ME ####',
  algoliaOptions: { 'facetFilters': ["version:$VERSION"] },
  debug: false // Set debug to true if you want to inspect the dropdown
});
</script>

- Add a search input in your page if you don't have any yet. Then update the inputSelector value in JS snippet
  to a CSS selector that targets your search input field.

- Replace $VERSION with the version you want to search on.
   The list of possible version is hardcoded in the config.
   So as of today you have: v0.1, v0.37, v0.5, v1.4

  For example if you want to refine the search to the version "v0.1" just specify: 'facetFilters': ["version:v0.1"]

- Optionally customize the look and feel by following the DocSearch documentation
  (See https://community.algolia.com/docsearch/styling.html)
- You can also check your configuration in our GitHub repository
  (See https://github.com/algolia/docsearch-configs/blob/master/configs/ory.json).

Please open a pull request if want to leverage your configuration!

Feel free to get back to us if you have any issues or questions regarding the integration.

We'd also be happy to get your feedback and thoughts about DocSearch - so we can continue to improve it.

Have a nice day :)

**Comment by aeneasr** [comment]
Thank you! It looks like the results have been updated! However, filtering for `tags` still doesn't work. If I use `facetFilters: ['tags:keto']` no results are returned, if I use `facetFilters: []` I do get results for the query `keto`. But since the tag is configured for the URL, shouldn't that work?

**Comment by s-pace** [comment]
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />


<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({
  apiKey: '8463c6ece843b377565726bb4ed325b0',
  indexName: 'ory',
  inputSelector: '### REPLACE ME ####',
  algoliaOptions: { 'facetFilters': ["tags:$TAGS", "version:$VERSION"] },
  debug: false // Set debug to true if you want to inspect the dropdown
});
</script>

- Add a search input in your page if you don't have any yet. Then update the inputSelector value in JS snippet
  to a CSS selector that targets your search input field.

- Replace $TAGS with the tags you want to search on.
   The list of possible tags is hardcoded in the config.
   So as of today you have: cloud, hydra, keto, kratos, oathkeeper

- Replace $VERSION with the version you want to search on.
   The list of possible version is hardcoded in the config.
   So as of today you have: v0.1, v0.37, v0.5, v1.4

  For example if you want to refine the search to the tags "cloud" and the version "v0.1" just specify: 'facetFilters': ["tags:cloud", "version:v0.1"]

**Comment by aeneasr** [comment]
Thank you! I figured out that I need to nest arrays when I want OR, e.g.: `[["tags:cloud", "tags:ecosystem"], "version:v0.1"]` :)

**Pull request merged**